core/templates/dev/head/pages/exploration_editor/ExplorationSaveAndPublishButtons.js

```diff
@@ -13,98 +14,520 @@
 // limitations under the License.
 
 /**
- * @fileoverview Controller for the exploration save & publish buttons.
+<<<<<<< HEAD
```

Comment:
Yes apologies, unsure how this merge conflict stayed in here. Illl remove the save and publish buttons.